Dockerfile

```diff
@@ -1,14 +1,15 @@
 FROM ubuntu:xenial as app
 
 RUN apt update && \
-  apt install -y git-core language-pack-en python3.5 python3-pip python3-dev libmysqlclient-dev libffi-dev libssl-dev build-essential gettext openjdk-8-jdk && \
-  pip3 install --upgrade pip setuptools && \
+  apt-get install -y software-properties-common && \
+  apt-add-repository -y ppa:deadsnakes/ppa && apt-get update && \
+  apt install -y git-core language-pack-en python3-pip python3.8-dev python3.8-venv libmysqlclient-dev libffi-dev libssl-dev build-essential gettext openjdk-8-jdk && \
```

Comment:
Now that we're using the virtualenv, I don't think you should need `python3-pip`.

Reply:
done